src/service/vaults.ts

```diff
@@ -155,6 +162,9 @@ const watchUserVaults = () => {
         [Kind.Data, path],
         value => {
           console.debug('got update', subscriber, value);
+          if (!value) {
+            return;
```

Comment:
We might receive the update to `offerIdsToPublicSubscribers` from the user's wallet and start watching the vault before the vault data is actually published to vstorage, so we don't want to error out in that case. Just wait for the data to show up assuming it will eventually. I'm not sure how else to handle it, but worst case if the contract had a bug the vault would just perpetually show a loading state.